IOS-HW14-Albums-Of-Pictures/ViewControllers/AlbumsViewController.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  AlbumsViewController.swift
+//  IOS-HW14-Albums-Of-Pictures
+//
+//  Created by temp user on 20.06.2022.
+//
+
+import UIKit
+
+class AlbumsViewController: UIViewController {
+    
+    lazy var collectionView: UICollectionView = {
+        
+        let collection = UICollectionView(frame: .zero, collectionViewLayout: createCompositionalLayout())
+        collection.translatesAutoresizingMaskIntoConstraints = false
+        collection.showsHorizontalScrollIndicator = false
+        collection.showsVerticalScrollIndicator = true
+        
+        collection.register(CustomCell.self, forCellWithReuseIdentifier: CustomCell.identifire)
+        collection.register(HeaderCustomSection.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HeaderCustomSection.identifier)
+        
+        collection.register(CustomSecondViewCell.self, forCellWithReuseIdentifier: CustomSecondViewCell.identifire)
+        collection.register(HeaderSecondSection.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HeaderSecondSection.identifier)
+        
+        collection.register(CustomThirdViewCell.self, forCellWithReuseIdentifier: CustomThirdViewCell.identifire)
+        collection.register(HeaderThirdSection.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HeaderThirdSection.identifier)
+        
+        collection.register(CustomFourthViewCell.self, forCellWithReuseIdentifier: CustomFourthViewCell.identifire)
+        collection.register(HeaderFourthSection.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: HeaderFourthSection.identifier)
+
+        
+        return collection
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .white
+        navigationController?.navigationBar.prefersLargeTitles = true
+        title = "Альбомы"
+        
+        configureAddButton()
+        setupCollectionView()
+    }
+    // Создание кнопки "+"
+    private func configureAddButton() {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            barButtonSystemItem: .add,
+            target: self,
+            action: nil
+        )
```

Comment:
Вполне можно было оставить на 47 строчка, да и в экшн добавить селектор :)
```suggestion
        navigationItem.leftBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(didTapped))
    }

    @objc func didTapped() {
        print("Нажата кнопка добавить")
    }
```